deploy/manifests/base/custom-types.yaml

```diff
@@ -104,6 +114,18 @@ spec:
           type: boolean
         config:
           type: object
```

Comment:
Please run the `./hack/build-single-manifests.sh` after any change to these YAMLs.